Hivemind/Assets/Scripts/ResourceNode.cs

```diff
@@ -63,7 +74,7 @@ private IEnumerator respawnResource()
     public int DecreaseFutureResources(int amount)
```

Comment:
ik kan deze comment niet zetten waar ik het wil, Regel 56-67 yield wait for 30 seconds is dit misschien beter om van dit aantal een serialize field te maken?